{{copier__project_slug}}/k8s/sandbox/postgres.cnpg.yaml

```
@@ -6,9 +6,43 @@ metadata:
     app: postgres
 spec:
   instances: 1
-  envFrom:
-  - configMapRef:
-      name: app-config
+  bootstrap:
+    initdb:
+      database: POSTGRES_DB
+      owner: POSTGRES_USER
```

We shouldn't set these. We can use the defaults generated from the cluster name and omit the bootstrap section. 
See https://cloudnative-pg.io/documentation/1.25/cloudnative-pg.v1/#postgresql-cnpg-io-v1-BootstrapInitDB for defaults.

Instead, we should add a commented section like:
```suggestion
  # TODO: Uncomment after first successful backup
  # bootstrap:
  #   recovery:
  #     source: clusterBackup
  externalClusters:
    - name: clusterBackup
      barmanObjectStore:
        destinationPath: "s3://{{ copier__project_dash }}-sandbox-backups/"
        # Copy backup from "postgres" to "postgres-restore"
        # Or set up a Kubernetes Job to copy it
        serverName: postgres-restore
        s3Credentials:
          accessKeyId:
            name: secrets-config
            key: AWS_S3_ACCESS_KEY_ID
          secretAccessKey:
            name: secrets-config
            key: AWS_S3_SECRET_ACCESS_KEY
        data:
          compression: gzip
        wal:
          compression: gzip
          maxParallel: 8
```